VogtPayroll5/Employee.cs

```diff
@@ -27,13 +27,25 @@ public decimal CalculateGrossPay(int hours, decimal rate)
             }
         }
 
+        public decimal CalculateOvertimePay(int hours, decimal rate)
```

Comment:
You shouldn't need to pass in the hours/rate if they're already class variables:
```c#
        public int HoursWorked { get; set; }
        public decimal HourlyPayRate { get; set; }
```

You should be able to access them directly from the object:
```c#
public decimal CalculateOvertimePay() {
            int  overtimeHours = 0;

            if (HoursWorked >= 40)
                overtimeHours = HoursWorked - 40;

            return overtimeHours * HourlyPayRate;
}
```